### PR TITLE
Release 28.0.6snap1

### LINKED ISSUE #2641: Update Nextcloud to 28
### Pending todos
- [x] Wait for 28.0.4 stable
- [x] https://github.com/nextcloud-snap/nextcloud-snap/pull/2665
- [x] Create branch `27`
- [x] Create label `27`

### Pending PRs
- [x] ~~https://github.com/nextcloud-snap/nextcloud-snap/pull/2657~~
- [x] https://github.com/nextcloud-snap/nextcloud-snap/pull/2655
- [x] https://github.com/nextcloud-snap/nextcloud-snap/pull/2659

### Pending upstream issues
- [x] https://github.com/nextcloud/server/issues/42286
- [x] https://github.com/nextcloud/server/issues/42671

### LINKED ISSUE #2658: [NC28] Configure maintenance time for daily resource intensive background jobs
grafik

Config to add ("1" stands for the first hour of the day, so 1 a.m.):

    'maintenance_window_start' => 1

Ref: https://docs.nextcloud.com/server/28/admin_manual/configuration_server/background_jobs_configuration.html#maintenance-window-start

### LINKED ISSUE #2734: MySQL starting with 8.0.37 requires cmake 3.11.2 or newer
MySQL 8.0.37 requires a newer version of cmake than what comes with core18.  Should attempt to build cmake snap or pull in from source then reference this newer version to make MySQL.

### LINKED ISSUE #2737: Enable dailies for Nextcloud 29
- [x] [Request `29` track](https://forum.snapcraft.io/t/track-29-for-nextcloud/40101)
- [ ] Setup channel on launchpad
- [x] Merge #2738
- [ ] Merge with next release to `master`

### LINKED ISSUE #2748: Update Nextcloud to 28.0.6
https://nextcloud.com/changelog/#latest28

### PR DESCRIPTION
**Pull request by Pilzinsel64**
Fix #2641 
Fix #2748
Fix #2737
Fix #2658
Fix #2734

### CODE REVIEW

**Review by scubamuc — APPROVED** [review]

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,12 @@
+v 28.0.6snap1
+  - nextcloud: update to 28.0.6
+  - php: update to 8.2.18
+  - redis: update to 7.0.15
+  - apache: update to 2.4.59
+  - ci: use cmake snap
```
Comment (pachulo) [inline CHANGELOG:6]:
```suggestion
  - mysql: use cmake snap to build
```

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ Nextcloud server packaged as a snap. It consists of:
 
 - Nextcloud 27
 - Apache 2.4
-- PHP 8.1
+- PHP 8.2
 - MySQL 8
 - Redis 6
```
Comment (pachulo) [inline README.md:9]:
```suggestion
- Redis 7
```

File: README.md [hunk 3]
```diff
@@ -4,7 +4,7 @@ Nextcloud server packaged as a snap. It consists of:
 
 - Nextcloud 27
```
Comment (pachulo) [inline README.md:5]:
```suggestion
- Nextcloud 28
```

**Comment by pachulo** [comment]
Does the changes that I proposed on the README belong to this PR or we need another one for that @kyrofa ? :thinking:

**Review by pachulo — APPROVED** [review]

File: CHANGELOG [hunk 4]
```diff
@@ -1,3 +1,12 @@
+v 28.0.6snap1
+  - nextcloud: update to 28.0.6
+  - php: update to 8.2.18
+  - redis: update to 7.0.15
+  - apache: update to 2.4.59
+  - mysql: use cmake snap to build
```
Comment (pachulo) [inline CHANGELOG:6]:
```suggestion
  - mysql: use cmake snap to build
  - mysql: update to 8.0.37
```

**Review by pachulo — APPROVED** [review]

**Review by scubamuc — APPROVED** [review]

**Comment by Pilzinsel64** [comment]
@pachulo @kyrofa It's now up to you to merge as only you have write access to `master`. :)

**Comment by pachulo** [comment]
Thanks for the hint @Pilzinsel64 , I just forgot about that :sweat_smile:
 
This is now building and should be on `latest/candidate` in one hour or so.

**Comment by scubamuc** [comment]
@pachulo and @Pilzinsel64 

there's mention that Readme wasn't upgraded see #2758

**Comment by Pilzinsel64** [comment]
Thanks! Just forgot it. Will be changed with the next update. :)

**Pull request merged**